src/main/java/gift/infra/EmployeeRepository.java

```diff
@@ -0,0 +1,45 @@
+package gift.infra;
+
+import gift.domain.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class EmployeeRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
```

Comment:
혹시 DI 이렇게 하는게 맞는지 궁금합니다!!!! 뭔가 다르게 했던 것 같은데 기억이 안나네요

Reply:
```suggestion
    @Autowired
private final JdbcTemplate jdbcTemplate;
```

만 하시면 됩니다. 어짜피 Spring에서 자동으로 생성자 주입을 해주기 때문에 아래 코드는 필요 없습니다

Reply:
아~~ 그렇군요!!! 강의 좀 다시 들으면서 공부좀 해야겠습니다 ㅜㅠㅠ 알려주셔서 감사합니다!!